### PR TITLE
Update prerequisites for preview.sh: corepack is required

### PR DESCRIPTION
**Pull request by lhotari**

### CODE REVIEW

File: contribute/document-preview.md [hunk 1]
```diff
@@ -18,9 +18,17 @@ Pulsar documentation is built using [Docusaurus](https://docusaurus.io/). To pre
...
 * Node >= 16.14
-* Yarn >= 1.5
+* Corepack installed and enabled (`corepack enable`)
 * Although you can use Linux, macOS, or Windows to build locally the Pulsar documentation, macOS is the preferred build environment as it offers the most complete support for documentation building.
 
+Installing prerequisites with [homebrew](https://brew.sh/) on MacOS or Linux:
+
+```shell
+brew install node
+brew install corepack
```
Comment (visortelle) [inline contribute/document-preview.md:28]: Not sure about the `brew install` step.

Corepack should be already available on newer Node without involving external package managers. Let me check.

Comment (visortelle) [reply contribute/document-preview.md:28]: Node.js v20.x.x (LTS)

Screenshot 2024-03-14 at 12 45 05 PM

Same for v21.x.x

Comment (lhotari) [reply contribute/document-preview.md]: Thanks, PTAL

**Review by visortelle — APPROVED** [review]
LGTM

**Pull request merged**